### PR TITLE
Docs: Update regarding new launcher

### PR DESCRIPTION
**Pull request by ReverM**
## What is this fixing or adding?
Updating world docs wr/ to the launcher changes introduced in 0.4.4
More precisely, clients are automatically installed with AP (you don't need to tick them during installation) and thus, you do not need to provide the roms on installation, only during patching/generation.

## How was this tested?
Reading

Here is a tasklist to keep track of which games need to be reviewed

- [ ] Archipelago Install guide
- [x] Adventure
- [x] ALTTP
- [x] Checksfinder
- [x] DKC3
- [x] FFMQ
- [x] LaDX
- [x] Lufia 2
- [x] Minecraft
- [ ] MMBN3
- [x] OOT
- [x] Super Metroid
- [x] Super Mario World
- [x] SMZ3
- [x] Undertale
- [x] Zillion

### CODE REVIEW

File: worlds/lufia2ac/docs/setup_en.md [hunk 1]
```diff
@@ -14,11 +14,12 @@ modded SNES minis are currently not supported by SNI**
 
 ## Installation Procedures
 
-1. Download and install SNIClient from the link above, making sure to install the most recent version.
-   **The installer file is located in the assets section at the bottom of the version information**.
-    - During setup, you will be asked to locate your base ROM file. This is your American Lufia II - Rise of the Sinistrals ROM file.
+1. Download and install [Archipelago](<https://github.com/ArchipelagoMW/Archipelago/releases/latest>). **The installer 
+   file is located in the assets section at the bottom of the version information.**
+2. The first time you do local generation or patch your game, you will be asked to locate your base ROM file. This is 
+   your American Lufia II - Rise of the Sinistrals ROM file. This only need to be done once.
```
Comment (lgwoodmark) [inline worlds/lufia2ac/docs/setup_en.md:20]: Typo: needs

File: worlds/smw/docs/setup_en.md [hunk 2]
```diff
@@ -23,9 +22,10 @@
 
 ### Windows Setup
 
-1. During the installation of Archipelago, you will have been asked to install the SNI Client. If you did not do this,
-   or you are on an older version, you may run the installer again to install the SNI Client.
-2. During setup, you will be asked to locate your base ROM file. This is your Super Mario World ROM file.
+1. Download and install [Archipelago](<https://github.com/ArchipelagoMW/Archipelago/releases/latest>). **The installer 
+   file is located in the assets section at the bottom of the version information.**
+2. The first time you do local generation or patch your game, you will be asked to locate your base ROM file. This is
+   your Super Mario World ROM file. This only need to be done once.
```
Comment (ReverM) [inline worlds/smw/docs/setup_en.md:28]:
```suggestion
   your Super Mario World ROM file. This only needs to be done once.
```

File: worlds/lufia2ac/docs/setup_en.md (hunk 1 above)
Comment (ReverM) [inline worlds/lufia2ac/docs/setup_en.md:20]:
```suggestion
   your American Lufia II - Rise of the Sinistrals ROM file. This only needs to be done once.
```

Comment (ReverM) [reply worlds/lufia2ac/docs/setup_en.md]: Fixed

**Comment by jonloveslegos** [comment]
Checksfinder and Undertale look fine.

**Review by t3hf1gm3nt — APPROVED** [review]
I know I'm not the maintainer for any of these games, but looked over at least the games I'm familiar with (ALTTP, DKC3, LADX, OOT, and SMW) and the changes for those games look good to me.

**Review by Alchav — APPROVED** [review]
Approve FFMQ section

**Review by beauxq — APPROVED** [review]
Zillion section LGTM

File: worlds/generic/docs/setup_en.md [hunk 3]
```diff
@@ -17,19 +17,22 @@ The most recent public release of Archipelago can be found on the GitHub Release
...
+
+- The server will allow you to host the multiworld on your machine. Hosting on your machine requires forwarding the port
 you are hosting on. The default port for Archipelago is `38281`. If you are unsure how to do this there are plenty of
 other guides on the internet that will be more suited to your hardware.
 
-The `Clients` are what are used to connect your game to the multiworld. If the game you plan to play is available
-here, go ahead and install its client as well. If the game you choose to play is supported by Archipelago but not listed
-in the installation, check the setup guide for that game. Installing a client for a ROM based game requires you to have
-a legally obtained ROM for that game as well.
+- The `Clients` are what are used to connect your game to the multiworld. Some games use a client that is automatically
```
Comment (ReverM) [inline worlds/generic/docs/setup_en.md:33]:
```suggestion
- The clients are what are used to connect your game to the multiworld. Some games use a client that is automatically
```
Potentially for uniformity.

File: worlds/generic/docs/setup_en.md [hunk 4]
```diff
@@ -17,19 +17,22 @@ The most recent public release of Archipelago can be found on the GitHub Release
...
+installed with an Archipelago installation. You can access those clients via the launcher or by navigating
+to your Archipelago installation.
```
Comment (BootsinSoots) [inline worlds/generic/docs/setup_en.md:35]: I would add a note that if they don't see a client, they will need to check the game page to learn how to set up the necessary connection to the multiworld

Comment (ReverM) [reply worlds/generic/docs/setup_en.md:35]: I chose not to because I believe every player should check the respective setup guide.

**Review by lordlou — APPROVED** [review]
I approove SM and SMZ3 changes.

**Comment by ThePhar** [comment]
I'm just going to take the executive decision to merge this so the website can get these updates since we're needing to update anyway.

File: worlds/generic/docs/setup_en.md (hunk 4 above)
Comment (ReverM) [reply worlds/generic/docs/setup_en.md:35]: Maybe a link could be added in the connecting section

**Review by ThePhar — APPROVED** [review]

**Pull request merged**